### ISSUE TITLE
obmc-console: load config file (/etc/obmc-console.conf?)

### ISSUE THREAD

**Issue by GHF** [openbmc/openbmc#937]
I was playing with the console server to get VUART to mirror to the hardware UART by adding `local-tty = ttyS0` to the config, which I assume is `/etc/obmc-console.conf`. However I don't think that the console server is actually loading that file.

Do we need to add `--config /etc/obmc-console.conf` to `obmc-console@.service` or something?

**Comment by williamspatrick** [comment]
Yeah, it looks like the obmc-console server doesn't load the config file by default.

https://github.com/openbmc/obmc-console/blob/master/console-server.c#L543

Need that option added to obmc-console@.service then.

We must be getting away with the defaults being good enough for the current needs.
